lib/builderator/interface/packer.rb

```diff
@@ -172,7 +172,7 @@ def _chef_execute_command(sudo = true)
       def _chef_install_command(sudo = true)
         template = sudo ? 'sudo ' : ''
         bash_cmd = "#{template}bash"
-        "#{bash_cmd} -a -c 'while #{template}fuser /var/lib/dpkg/lock >/dev/null 2>&1; do sleep 5; done; curl -L https://www.chef.io/chef/install.sh' | #{bash_cmd} -s -- -v #{Config.chef.version}"
+        "flock /var/lib/dpkg/lock -c 'curl -L https://www.chef.io/chef/install.sh | #{bash_cmd} -s -- -v #{Config.chef.version}'"
```

Comment:
One thing you _may_ run into (you could test this locally) is that ruby might not render your variables inside single quotes.  So your string may be taken literally.  just watch out for that in testing.